db/migrate/20230727102149_create_training_module_records.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTrainingModuleRecords < ActiveRecord::Migration[7.0]
```

Comment:
I think it would be a useful exercise to describe in simple terms why we are introducing a new table. 

1. Why was it proposed?
2. What value should it add?
3. What could it do for us in the future that we need be aware of?

We know we only have limited info in the hook payload so we have to query what has changed. 

Is that potentially a lengthy process? 
What sort of methods on our objects would make this easier, do we have everything all ready?
Will it be possible to make a custom API call and retrieve details from Launch?

A little pseudo-code might aid the discussion.

Reply:
I've added some comments here with my thoughts. Can I clarify what you mean by launch?